ReactSkia/components/RSkComponentTextInput.cpp

```diff
@@ -242,6 +255,7 @@ void RSkComponentTextInput::onHandleKey(rnsKey eventKeyType, bool keyRepeat, boo
         inputQueueMutex.unlock();
         eventCount_++;
 	resignFromEditingMode();
+	OnScreenKeyboard::exit();
```

Comment:
can we handle OSK exit in resignFromEditingMode itself ? than adding this call at multiple places

Reply:
Yes, it will be good to move into resignFromEditingMode . Will update it.